### PR TITLE
Export Seed Phrase

### PR DESCRIPTION
**Pull request by itzmeanjan**
- Guide for exporting seed phrase from Matic Mobile Wallet.
- Build issue resolved.

cc @vidyaakrishna @ryegineni

### CODE REVIEW

**Review by anurag-arjun — APPROVED** [review]

File: sidebars.js [hunk 1]
```diff
@@ -89,6 +89,7 @@ module.exports = {
       label: "Wallets",
       items: [
         "develop/wallets/getting-started",
+        "develop/wallets/exporting-seed-phrase",
```
Comment (anurag-arjun) [inline sidebars.js:92]: Put it under a new category `Matic Wallet`

**Review by anurag-arjun — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 1 above)
Comment (itzmeanjan) [reply sidebars.js:92]: Ok, doing so.

Comment (itzmeanjan) [reply sidebars.js]: Done.

Comment (itzmeanjan) [reply sidebars.js]: Conflict resolved.

**Review by anurag-arjun — APPROVED** [review]

**Pull request merged**